package.nls.json

```diff
@@ -8,9 +8,18 @@
 "mssql.scriptDelete":"Script as Drop",
 "mssql.scriptExecute":"Script as Execute",
 "mssql.scriptAlter":"Script as Alter",
+"mssql.openQueryHistory":"Open Query",
+"mssql.runQueryHistory":"Run Query",
+"mssql.deleteQueryHistory":"Delete",
+"mssql.clearAllQueryHistory":"Clear All History",

```

Comment:
we seem to say "Query History" everywhere else, but just say "History" here.  I guess from the context in the UI it is clear that this text applies only to query history?

Reply:
I got the text from ADS for it, but you make a good point. I'll add `Query History` for the command palette text, and let the context menu action string be the same.